### PR TITLE
[Migration] Asset conversion tutorial from Polkadot Wiki

### PR DESCRIPTION
**Pull request by nhussein11**
This pull request involves migrating the Asset Conversion tutorial from the Polkadot Wiki to our documentation. The tutorial covers creating liquidity pools, adding liquidity, and performing token swaps using the Asset Conversion pallet. 

Polkadot wiki link: https://wiki.polkadot.network/docs/learn-guides-asset-conversion 

Please note that there may be redundant and similar sections on the Polkadot wiki. Therefore, I suggest splitting this page into the following:

- asset-conversion-introduction.md
- creating-a-liquidity-pool.md
- adding-liquidity-to-a-pool.md
- swapping-assets.md
- withdrawing-assets.md

### CODE REVIEW

File: docs/build-on-polkadot/parachains/asset-hub/image-1.png
Comment (0xLucca) [inline docs/build-on-polkadot/parachains/asset-hub/image-1.png:1]: What are these images?

File: docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+- Creating a liquidity pool
+- Adding liquidity to a pool
+- Swapping assets
+- Withdrawing liquidity from a pool
+
+## Prerequisites
+
+Before converting assets on Asset Hub, you must ensure you have:
+
+- Access the [Polkadot.Js App](https://polkadot.js.org/apps){target=\_blank} interface and connect it to the intended blockchain
```
Comment (0xLucca) [inline docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md:24]:
```suggestion
- Access the [Polkadot.ss Apps](https://polkadot.js.org/apps){target=\_blank} interface and connect it to the intended blockchain
```

File: docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+To add liquidity to a pool, follow these steps:
+
+1. Navigate to the **Extrinsics** section on the Polkadot.Js App interface
+      1. Select **Developer** from the top menu
+      2. Click on **Extrinsics** from the dropdown menu
+
+        ![Extrinsics Section](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-1.webp)
+
+2. Choose the **AssetConversion** pallet and click on the **add_liquidity** extrinsic
+      1. Select the **AssetConversion** pallet
```
Comment (0xLucca) [inline docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md:107]:
```suggestion
2. Choose the **assetConversion** pallet and click on the **add_liquidity** extrinsic
      1. Select the **assetConversion** pallet
```

File: docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+         ```
+      5. Click on **Submit Transaction** to create the liquidity pool
+
+        ![Create Pool Fields](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-3.webp)
+
+After signing and submitting the transaction, the liquidity pool will be created. To verify the pool’s creation, check the **Explorer** section on the Polkadot.Js App interface and ensure that the **PoolCreated** event has been emitted.
+
+![Pool Created Event](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-4.webp)
+
+As the above image shows, the **lp_token** ID created for this pool is 19. This ID is essential to identify the liquidity pool and the LP tokens associated with it.
```
Comment (0xLucca) [inline docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md:92]:
```suggestion
As the above image shows, the **lpToken** ID created for this pool is 19. This ID is essential to identify the liquidity pool and the LP tokens associated with it.
```

File: docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+
+        In this case, the liquidity provided to the pool is between DOT tokens and PPM tokens with the asset ID 1112 on Polkadot Asset Hub. The intention is to provide liquidity for 1 DOT token (u128 value of 1000000000000 as it has 10 decimals) and 1 PPM token (u128 value of 1000000000000 as it also has 10 decimals).
+
+After signing and submitting the transaction, the liquidity will be added to the pool. To verify the liquidity addition, check the **Explorer** section on the Polkadot.Js App interface and ensure that the **LiquidityAdded** event has been emitted.
+
+![Liquidity Added Event](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-7.webp)
+
+## Swapping Assets
+
+### Swapping From an Exact Amount of Tokens
```
Comment (0xLucca) [inline docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md:149]: The two images have different extrinsics:
one has **swapTokensForExactTokens** and the other has **swapExactTokensForTokens**

File: docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+
+1. Navigate to the **Extrinsics** section on the Polkadot.Js App interface
+      1. Select **Developer** from the top menu
+      2. Click on **Extrinsics** from the dropdown menu
+
+        ![Extrinsics Section](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-1.webp)
+
+2. Choose the **AssetConversion** pallet and click on the **swapForExactTokens** extrinsic
+      1. Select the **AssetConversion** pallet
+      2. Choose the **swapForExactTokens** extrinsic from the list of available extrinsics
```
Comment (0xLucca) [inline docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md:163]:
```suggestion
      2. Choose the **swapExactTokensForTokens** extrinsic from the list of available extrinsics
```

File: docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+
+        ![Swap For Exact Tokens Extrinsic](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-8.webp)
```
Comment (0xLucca) [inline docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md:165]:
```suggestion
        ![Swap From Exact Tokens Extrinsic](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-8.webp)
```

File: docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md [hunk 7]
```diff
@@ -0,0 +1,321 @@
...
+        ![Swap For Exact Tokens Fields](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-9.webp)
+
+        !!! warning
+            Before swapping assets, ensure that the tokens provided have been minted previously and are available in your account.
+
+        In this case, the intention is to swap 0.01 DOT token (u128 value of 100000000000 as it has 10 decimals) for 0.04 PPM token (u128 value of 400000000000 as it also has 10 decimals).
+
+After signing and submitting the transaction, the swap will be executed. To verify the swap, check the **Explorer** section on the Polkadot.Js App interface and ensure that the **SwapExecuted** event has been emitted.
+
+![Swap For Exact Tokens Event](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-10.webp)
```
Comment (0xLucca) [inline docs/build-on-polkadot/parachains/asset-hub/asset-conversion.md:200]:
```suggestion
![Swap From Exact Tokens Event](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/asset-conversion/asset-conversion-10.webp)
```

File: docs/build-on-polkadot/parachains/asset-hub/image-1.png
Comment (nhussein11) [reply docs/build-on-polkadot/parachains/asset-hub/image-1.png:1]: seems that they got leaked from our snagit template hahah 🙃

File: docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md [hunk 8]
```diff
@@ -46,13 +46,13 @@ This guide will use Polkadot, its local Asset Hub instance, and the [Astar](http
       1. Click on the **Developer** tab from the top navigation bar
       2. Select **Extrinsics** from the dropdown
 
-    ![Access to Developer Extrinsics section](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-1.webp)
+    ![Access to Developer Extrinsics section](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-1.webp)
 
 3. Select the Foreing Assets pallet
```
Comment (dawnkelly09) [inline docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md:51]:
```suggestion
3. Select the Foreign Assets pallet
```

File: docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md [hunk 9]
```diff
@@ -46,13 +46,13 @@ This guide will use Polkadot, its local Asset Hub instance, and the [Astar](http
...
       3. Select the **foreignAssets** pallet from the dropdown list
```
Comment (dawnkelly09) [inline docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md:52]:
```suggestion
      3. Select the **`foreignAssets`** pallet from the dropdown list
```

File: docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md [hunk 10]
```diff
@@ -46,13 +46,13 @@ This guide will use Polkadot, its local Asset Hub instance, and the [Astar](http
...
       4. Choose the **create** extrinsic
```
Comment (dawnkelly09) [inline docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md:53]:
```suggestion
      4. Choose the **`create`** extrinsic
```

File: docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md [hunk 11]
```diff
@@ -69,12 +69,12 @@ This guide will use Polkadot, its local Asset Hub instance, and the [Astar](http
 
             Ensure that **Sibling** is selected and that the **Para ID** corresponds to the source parachain. In this case, since the guide follows the test setup stated in the [Test Environment Setup](#test-environment-setup) section, the **Para ID** is `2006`.
 
-            ![Get parachain sovereign account](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-3.webp)
+            ![Get parachain sovereign account](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-3.webp)
 
 
     - **minBalance** - the minimum balance required to hold this asset
```
Comment (dawnkelly09) [inline docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md:75]:
```suggestion
    - **`minBalance`** - the minimum balance required to hold this asset
```

**Review by dawnkelly09 — CHANGES_REQUESTED** [review]
Formatting and grammar suggestions. Thank you!

**Review by dawnkelly09 — CHANGES_REQUESTED** [review]
Formatting and grammar suggestions. Thank you!

**Comment by albertov19** [comment]
@nhussein11 what is the status of this PR? Dawn left some comments as well as Vale.

Should we review this one? Or you ported this changes to the new repo?

**Comment by nhussein11** [comment]
> @nhussein11 what is the status of this PR? Dawn left some comments as well as Vale.
> 
> Should we review this one? Or you ported this changes to the new repo?

@albertov19, I have addressed all of Dawn's comments. As for Vale's comments, the remaining ones are false positives. I think this PR is ready to be merged. After that, it should be ported to the new repository since this content is not there yet.

**Pull request merged**